cli/internal/lockfile/pnpm_lockfile.go

```diff
@@ -186,6 +186,19 @@ func (p *PnpmLockfile) Subgraph(workspacePackages []turbopath.AnchoredSystemPath
 		return nil, err
 	}
 
+	for _, importer := range importers {
+		for dependency, meta := range importer.DependenciesMeta {
+			if meta.Injected {
+				resolution, ok := importer.Dependencies[dependency]
```

Comment:
Here it's also necessary to lookup the `importer.DevDependencies`, because `injected` dependency cat also be a devDependency